charts/prefect-server/values.yaml

```diff
@@ -53,10 +53,16 @@ server:
   # -- priority class name to use for the server pods; if the priority class is empty or doesn't exist, the server pods are scheduled without a priority class
   priorityClassName: ""
 
-  # -- enable server debug mode
+  # ref: https://docs.prefect.io/v3/develop/settings-ref#base-path
```

Comment:
added additional docs ref's for specific server environment variables. Also removed double `##`